### DISCUSSION TITLE
DM1702 status?

### DISCUSSION THREAD

**Discussion by mr-sm1th** [hmatuschek/qdmr#199] [Q&A]
i noticed you have a build branch for this radio,
what is the status? i have one if you need help?

**Comment by allesand** [comment]
Also ref in #181

**Comment by darkman738** [comment]
Would be great if there were movement here.  It seems this is an under supported radio.  It's difficult to find any app that works with it on any OS.  I've only found 1 for Windows, and it's quite a pain.  Happy to test things locally!
